pkg/orchestrator/callback.go

```diff
@@ -76,7 +76,7 @@ func (e *Callback) OnBidComplete(ctx context.Context, response legacy.BidResult)
 		return
 	}
 
-	if err = e.store.AddExecutionHistory(txContext, response.JobID, response.ExecutionID, executionEvents...); err != nil {
+	if err = e.store.AddExecutionHistory(txContext, response.JobID, response.JobVersion, response.ExecutionID, executionEvents...); err != nil {
```

Comment:
THink about the backward compatibility and is this callback even called or just for legacy compute nodes ?

Reply:
Added warning